about.html

```diff
@@ -10,7 +10,7 @@ <h1 class="mobile-only title">The {{ site.github.owner_name }} Group</h1>
     <div class="maintainer col-12 col-md-4 col-lg-3" id="{{ maintainer.login }}">
       <a href="{{ maintainer.html_url}}">
         <div class="card">
-          <img class="maintainer-img card-img-top" loading="lazy" src="{{ maintainer.avatar_url }}&size=291">
+          <img class="maintainer-img card-img-top" loading="lazy" src="{{ maintainer.avatar_url }}&size=291 alt=""">
```

Comment:
alt is inside src here

Reply:
I have learned copy already, but not paste... :facepalm:

Reply:
![](https://user-images.githubusercontent.com/3535780/163846660-e9acab29-783e-4b42-b0c8-9560a2068d13.png)